index.html

```diff
@@ -0,0 +1,213 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="twitter.css" />
+    <script
+      src="https://kit.fontawesome.com/6e6e2bbd4d.js"
+      crossorigin="anonymous"
+    ></script>
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
+    />
+    <title>Static Template</title>
+  </head>
+  <body>
+    <div
+      class="three-column-grid__expand-two gap-sm center-wrapper main-layout"
+    >
+      <header class="main-header three-row-grid__expand-two gap-sm">
+        <aside class="Side-bar-menu">
+          <div>
+            <img
+              class="twitter-logo"
+              src="https://img.search.brave.com/jaw-9w6G39GIIxdKdqgSGOIdknwh344YLISmmJE_2rw/rs:fit:1200:1200:1/g:ce/aHR0cHM6Ly9sb2dv/cy1kb3dubG9hZC5j/b20vd3AtY29udGVu/dC91cGxvYWRzLzIw/MTYvMDIvVHdpdHRl/cl9Mb2dvX25ldy5w/bmc"
+              alt="Twitter Logo"
+            />
+          </div>
+          <nav>
+            <ul class="menu-list">
+
+              <li class="active">
```

Comment:
nice use of an active class, makes it really quick and simple to add or remove this property